token.go

```diff
@@ -34,6 +34,8 @@ type TokenCreateRequest struct {
 type Token struct {
 	KeystoneToken string `json:"token"`
 	ExpiresAt     string `json:"expires_at"`
+	ProjectName   string `json:"tenant_name"`
```

Comment:
please check with auth service, return  project_name in response payload  instead of tenant name./